tests/integration/query/simple/with_operation_name_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package simple
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestQuerySimpleWithMultipleOperationsAndOperationName_ShouldSucceed(t *testing.T) {
```

Comment:
todo: Please do not create new tests using the old `RequestTestCase` - please use `TestCase`